core/src/main/java/org/apache/iceberg/util/SnapshotUtil.java

```diff
@@ -63,6 +63,19 @@ public static boolean ancestorOf(Table table, long snapshotId, long ancestorSnap
     return ancestorIds(table.currentSnapshot(), table::snapshot);
   }
 
+  /**
+   * Traverses the history of the table's state and finds the oldest Snapshot.
+   * @return null if the table is empty, else the oldest Snapshot.
```

Comment:
What does "empty" mean? I think it means that there is no current snapshot, but it would be better to say that.

Reply:
yeah, that makes it clearer.. fixed it.